docs/advanced-features/codemods.md

```diff
@@ -64,7 +64,97 @@ Dangerously migrates from `next/legacy/image` to the new `next/image` by adding
 - Removes `objectPosition` prop and adds `style`
 - Removes `lazyBoundary` prop
 - Removes `lazyRoot` prop
-- TODO: handle `loader`
+- TODO: does not migrate the `loader` config. If you need it, you must manually add a `loader` prop.
```

Comment:
Note this TODO was merged

Reply:
Yep, this is not implemented yet

Reply:
@leerob This is implemented in PR #41633